README.md

```diff
@@ -1,3 +1,34 @@
 # Augur
 
-This repository will serve as the central location for Augur Issues and other items.
+## Define: Augur Protocol
+Augur is a decentralized oracle and peer to peer protocol for prediction markets. Augur is free, public, open source software, portions of which are licensed under the General Public License (GPL) and portions of which are licensed under the Massachusetts Institute of Technology (MIT) license. Augur is a set of smart contracts written in Solidity that can be deployed to the Ethereum blockchain.
+
+Augur is a protocol, freely available for anyone to use however they please. Augur is accessible through a desktop client app, similar to interacting with an Ethereum or Bitcoin node. Users of the Augur protocol must themselves ensure that the actions they are performing are compliant with the laws in all applicable jurisdictions and must acknowledge that others’ use of the Augur protocol may not be compliant. Users of the Augur protocol do so at their own risk.
+
+For more information about the Augur protocol and the Forecast Foundation, [check out the FAQ](https://www.augur.net/faq/).
+
+## Want to try Augur?
+
+[Grab the latest release of Augur App](https://github.com/AugurProject/augur-app/releases) to connect to the Ethereum Network and explore what the Augur Community has done.
+
+## 100% Fully Open Source
+
+You can view, edit, and contribute to Augur via our 100% open source repositories hosted on GitHub!
+
+- [Augur Core](https://github.com/AugurProject/augur-core) - The core implementation of the Augur Project as Smart Contracts written in Solidity for the Ethereum Network.
+- [Augur.js](https://github.com/AugurProject/augur.js) - Javascript library for Node.js and the browser to help you communicate with the Augur smart contracts, and Augur Node.
+- [Augur Node](https://github.com/AugurProject/augur-node) - Helper application to aggregrate the activity of the Augur Community on the Ethereum Network. Allows you to quickly search and fitler markets, and build responsive and performant user interfaces.
+- [Augur UI](https://github.com/AugurProject/augur-ui/) - A reference client UI which uses Augur Node and a connection to the Ethereum Network to interact with the Augur community.
+- [Augur App](https://github.com/AugurProject/augur-app/) - An electron-based application which combines the above projects in an easy-to-use intallable application.
+
+## Documentation and Whitepaper
+
+Developer documentation is available on [the Augur documentation site](https://docs.augur.net/). If you find any mistakes or want to add any clarifications, you can submit pull requests for updates to [the github repository](https://github.com/AugurProject/docs).
+
+If you're interested in going in-depth with Augur, [The Augur Whitepaper](https://github.com/AugurProject/whitepaper) provides an in-depth look at the game theory behind the Augur Protocol.
```

Comment:
Suggest:
- `[The Augur Whitepaper](https://github.com/AugurProject/whitepaper)` -> `the Augur [whitepaper](https://github.com/AugurProject/whitepaper)`
- `Augur Protocol` -> `Augur protocol`